### ISSUE TITLE
Report Errors at bottom of output on pulumi up

### ISSUE THREAD

**Issue by kvarforl** [pulumi/pulumi#11972]
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details

Would it be possible to have the summary section (or whatever is at the bottom) show some indication that there were errors to look into further up in the output? 

This or some other very blatant reporting of errors would improve my workflow considerably :)

I frequently run `pulumi up` commands in short terminals (ie, the integrated terminal in the bottom pane of VS Code or Jetbrains rider). Because of this, the diagnostics get printed off screen when there is an error. I also regularly start a pulumi up, work on something else, and then come back to see if it completed: often the first thing / only thing I see is the Summary section, which tells me how many things were updated, created, and deleted, but doesn't report errors. 

Other issues that are semi related but not the same: [9231](https://github.com/pulumi/pulumi/issues/9231), [1601](https://github.com/pulumi/pulumi/issues/1601),

**Comment by Frassle** [comment]
Sounds reasonable, I'd hope this would be fairly simple to add.
cc @dixler and @susanev as y'all usually have comments for layout things like this.

**Comment by Manishpandey11** [comment]
Hi,
I want to work on this issue. Can you please assign it to me

**Comment by susanev** [comment]
yes, we should absolutely do this. happy to provide feedback once a pr is up.

**Comment by croddin** [comment]
I have this issue as well, The section showing the errors doesn't necessarily need to move lower. I would say either move the errors nearer to the bottom, add an errors count in red to the resources summary, or have the last line always say either "deployment successful" in green or "deployment had errors" in red or something. Maybe a combination of several of those.

**Comment by ashish111333** [comment]
@Frassle  looking at this again based on the above comment  I am going to add  a short line at the end of Resource sec indicating the total errors.
